synthetics-client/scheduler.py

```diff
@@ -16,11 +20,20 @@
 import yaml
 import time
 
+TIME_BETWEEN_SUITE_CHECK = 60
```

Comment:
Np, I noticed there're some hardcoded values. You may consider adding a config file to load from

Reply:
Pratik and I are considering whether to have this value be set through command line arguments or a config file, what do you believe is the better option?

Reply:
Even if you set it through cmd, unless you want to make it a required value, otherwise it will still need a default value which shouldn't be hard-coded. I'd say if you have other values that you need to set for the scheduler, use a config file

Reply:
@zhongnansu I'll make this a configurable option and remove all hard-coded values in the next PR. 
Thanks for the catch!